Sources/JsonModel/ResultData/ResultData.swift

```diff
@@ -1,7 +1,7 @@
 //
 //  ResultData.swift
 //
-//  Copyright © 2020 Sage Bionetworks. All rights reserved.
+//  Copyright © 2017-2021 Sage Bionetworks. All rights reserved.
```

Comment:
2017 or 2020 (previous copyright)?

Reply:
This file now includes the *protocol* as well as an object that conforms to that protocol. The protocol was written in 2017.